client/src/components/dashboard.js

```diff
@@ -146,28 +131,19 @@ class Dashboard extends Component {
                     <Skeleton active/>
                 </Card>
             </Col>)
-        };
+        }
 
         return (
             <div>
-
-                <Dropdown overlay={
-                    <Menu>
-                        {this.state.projects.map( (value, index) => {
-
-                            return (<Menu.Item
-                                key={index} >
-                                {/* The title value is returned in the second index*/}
-                                {value[1]}
-
-                            </Menu.Item>)
-                        })
-                        }
-                    </Menu>
-
-                } placement="bottomCenter">
-                    <Button> Public Projects</Button>
-                </Dropdown>
+                {
+                    this.state.networkError ?
+                    <Alert
+                    message="Network Error"
+                    description={"Please point to any of these networks: " + deployment.getActiveNetworks().map((network) => { return (network.network_name)})}
```

Comment:
`... => network.network_name` should work. No need for a return keyword. Also, the camel casing is preferred in javascript.

Reply:
I will resolve this. Since it was a config left it non-camel case. But we can still change this to JS standard.